lib/rules/max-nesting-depth/__tests__/index.js

```diff
@@ -110,6 +110,38 @@ testRule(rule, {
   ]
 });
 
+testRule(rule, {
+  ruleName,
+  config: [1, { ignore: ["blockless-pseudoclasses"] }],
```

Comment:
I believe [the word is hyphenated](https://www.w3.org/TR/selectors-4/#pseudo-classes) e.g. `"blockless-pseudo-classes"`. Please edit in the documentation too.